internal/mock/sqladmin.go

```diff
@@ -229,3 +249,40 @@ func CreateEphemeralSuccess(i FakeCSQLInstance, ct int) *Request {
 	}
 	return r
 }
+
+// CreateEphemeralSuccess returns a Request that responds to the
+// `sslCerts.createEphemeral` SQL Admin endpoint. It responds with a "StatusOK" and a
+// SslCerts object.
+//
+// https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/sslCerts/createEphemeral
+func CreateEphemeralSuccess(i FakeCSQLInstance, ct int) *Request {
+	return CreateEphemeralSuccessWithExpiry(i, ct, time.Now().Add(time.Hour))
+}
+
+// TestClient is a convenience wrapper that configures a mock HTTP client and
+// sqladmin.Service based on a connection name.
+func TestClient(proj, reg, name string, db *sqladmin.DatabaseInstance, certExpiry time.Time) (*sqladmin.Service, func() error, error) {
+	inst := NewFakeCSQLInstance(proj, reg, name)
+	mc, url, cleanup := HTTPClient(
+		InstanceGetSuccessWithDatabase(inst, 1, db),
+		CreateEphemeralSuccessWithExpiry(inst, 1, certExpiry),
+	)
+	client, err := sqladmin.NewService(
+		context.Background(),
+		option.WithHTTPClient(mc),
+		option.WithEndpoint(url),
+	)
+	return client, cleanup, err
+}
+
+// genRSAKey generates an RSA key used for test.
+func genRSAKey() *rsa.PrivateKey {
+	key, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		panic(err) // unexpected, so just panic if it happens
+	}
+	return key
+}
+
+// RSAKey is used for test only.
+var RSAKey = genRSAKey()
```

Comment:
nit: In general, I think globals should try to stay towards the top of the file.

That said, this seems like a regression to me - if each instance has it's own key, they intentionally cant be mixed up